.github/workflows/gradle-build.yml

```diff
@@ -51,3 +51,36 @@ jobs:
             gradle.properties
             **/gradle/**
             Scripts/**
+  
+  compare-packages:
+    needs: build 
+    runs-on: ubuntu-latest
+    env: 
+      CURRENT_PKG_DIFF: ./artifacts/current/drop/build/libs/msgraph-beta-sdk-java.jar 
+      PRIOR_PKG_DIFF: ./artifacts/previous/drop/build/libs/msgraph-beta-sdk-java.jar
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Download Current Build
+        uses: actions/download-artifact@v2
+        with: 
+          name: drop
+          path: artifacts/current/
+      - name: Download Last Successful Build
+        uses: dawidd6/action-download-artifact@v2.14.1
+        with: 
+          workflow: preview-and-release.yml
+          workflow_conclusion: success
+          branch: dev
+          event: push
+          name: drop
+          path: artifacts/previous/
+      - name: Run PKG Diff 
```

Comment:
you might need to install java, because pkgdiff relies on java being available as well